app/components/kb/relevant-articles.vue

```diff
@@ -2,93 +2,79 @@
     <div>
         <slot name="title">
             <h4>
-            {{ $t("relevantArticles") }} 
-                <hr></hr>
+            {{ t("relevantArticles") }}
+                <hr/>
             </h4>
         </slot>
-        <div class="loading" v-if="loading"><i class="fa fa-cog fa-spin fa-lg" ></i> {{ $t("loading") }}...</div>
+        <div class="loading" v-if="loading"><i class="fa fa-cog fa-spin fa-lg" ></i> {{ t("loading") }}...</div>
         <ul>
             <li v-for="article in articles" class="mb-1">
                 <a class="link-dark fs-6" :href="`${articleUrl(article)}`">{{lstring(article.title, $locale)}}</a>
             </li>
         </ul>
         <div v-if="articles.length<1 && !loading" class="alert alert-light">
-            <strong>{{ $t("noResultFound") }}</strong>
+            <strong>{{ t("noResultFound") }}</strong>
         </div>
     </div>
-</template>
-
-<script>
-import ArticlesApi from './article-api';
-import i18n from '../../app-text/components/kb.json';
-import articlesMaxin from '../maxin/article';
-import { lstring } from './filters';
-export default {
-    name: 'kbRelevantArticles',
-    components: {},
-    props: {
-        tag: String,
-        type: String,
+  </template>
+  
+<script setup>
+    import { ref, onMounted } from "vue";
+    import ArticlesApi from "./article-api";
+    import { getUrl } from '../../services/composables/articles.js';
+    import { lstring } from "./filters";
+    import { useRealm } from '../../services/composables/realm.js';
+    import { useI18n } from 'vue-i18n';
+    import messages from '../../app-text/components/kb.json';
+    const { t } = useI18n({ messages });
+    const realm = useRealm();
+    const { tag, type, sort } = defineProps({
```

Comment:
destructing the props will loose the reactivity, use 
```suggestion
    const props = defineProps({
```

Reply:
updated as suggested.